kuma/wiki/utils.py

```diff
@@ -64,7 +64,11 @@ def get_doc_components_from_url(url, required_locale=None, check_host=True):
     # Extract locale and path from URL:
     parsed = urlparse(url)  # Never has errors AFAICT
     if check_host and parsed.netloc:
-        return False
+        # If its not redirecting to our domain,
+        # It should not be our wiki document
```

Comment:
Nit: "It" should not be capitalized, since it doesn't start a new sentence. This would fit on one line, avoiding the issue:

```
# Only allow redirects on our domain
```